app/controllers/admin/edition_tags_controller.rb

```diff
@@ -16,6 +16,19 @@ def update
       previous_version: params["taxonomy_tag_form"]["previous_version"],
     )
 
+    # Only automatically add legacy associations if this is new
+    # content, to avoid repeatedly adding legacy associations if
+    # they've been manually removed
+    unless @edition.published?
```

Comment:
Is this maybe better moved into EditionTaxonLinkPatcher? That would keep logic out of this and also applies the taggings to the Statistics Announcements case.

Reply:
Well, given that Statistics Announcements are not editions in the Whitehall sense, I find it a bit odd that the EditionTaxonLinkPatcher is used...

Also, with the current flow for Statistics Announcements, the mapping stuff won't really work (as the policy areas appear prior to the taxonomy tagging interface). I think we'll have to work out what to do with the wider problem at some point, but just in terms of the mapping stuff, I think the thing to do for now is consider it out of scope.